<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>541</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>541</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JSON Parser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(28, 28, 28);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="file_path_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 2px solid rgb(2, 210, 180);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
color: rgb(255, 255, 255);
font: 13pt &quot;Arial Rounded MT&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 2px solid rgb(2, 210, 180);
background-color: rgb(85, 255, 127);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
background-color: rgb(2, 210, 180);
color: rgb(255, 255, 255);
font: 15pt &quot;Arial Rounded MT&quot;;
</string>
    </property>
    <property name="text">
     <string>Открыть JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_file_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 2px solid rgb(2, 210, 180);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
background-color: rgb(2, 210, 180);
color: rgb(255, 255, 255);
font: 15pt &quot;Arial Rounded MT&quot;;
</string>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QLabel" name="check_result_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>161</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
border-bottom: 2px solid rgb(2, 210, 180);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
color: rgb(255, 255, 255);
font: 15pt &quot;Arial Rounded MT&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 2px solid rgb(2, 210, 180);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
background-color: rgb(2, 210, 180);
color: rgb(255, 255, 255);
font: 15pt &quot;Arial Rounded MT&quot;;
</string>
    </property>
    <property name="text">
     <string>Путь к файлу</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>451</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(2, 210, 180);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>435</width>
       <height>369</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLabel" name="json_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>361</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT</family>
        <pointsize>17</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
border-bottom: 2px solid rgb(2, 210, 180);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
color: rgb(255, 255, 255);
font: 17pt &quot;Arial Rounded MT&quot;;
}</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="error_header">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 2px solid rgb(2, 210, 180);
border-top: 2px solid rgb(2, 210, 180);
border-right: 2px solid rgb(2, 210, 180);
border-left: 2px solid rgb(2, 210, 180);
border-radius:8px;
background-color: rgb(2, 210, 180);
color: rgb(255, 255, 255);
font: 15pt &quot;Arial Rounded MT&quot;;
</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
